store/postgres/migrations/20211107162613_create_users_table.up.sql

```diff
@@ -0,0 +1,9 @@
+CREATE TABLE IF NOT EXISTS users (
+  id uuid PRIMARY KEY DEFAULT uuid_generate_v4(),
+  name varchar UNIQUE NOT NULL,
+  email varchar UNIQUE NOT NULL,
+  metadata jsonb,
+  created_at timestamptz NOT NULL DEFAULT NOW(),
+  updated_at timestamptz NOT NULL DEFAULT NOW(),
+  deleted_at timestamptz
+);
```

Comment:
new line in the end 😅 , as that's how a file is defined, ending with a empty line.
Ref: https://pubs.opengroup.org/onlinepubs/9699919799/basedefs/V1_chap03.html#tag_03_206